### DISCUSSION TITLE
SVG content is changed in NextJS dist/static/images directory

### DISCUSSION THREAD

**Discussion by forevereffort** [webpack/webpack#12704] [Q&A]
I am developing the project by NextJS SSR mode. I got the problem so the svg content is changed in the static directory like the following. How can I fix this issue?

![JAxL9](https://user-images.githubusercontent.com/58469967/108305434-eac20c80-71e4-11eb-9d76-8031fdb4ce65.png)

Regards

**Comment by alexander-akait** [comment]
Please report the problem in `NextJS` repo
